M2/Macaulay2/m2/nets.m2

```diff
@@ -220,40 +220,36 @@ alignmentFunctions := new HashTable from {
 netList VisibleList := o -> (x) -> (
```

Comment:
It would be cool if negative numbers could be used to indicate offsets from the end of the list, i.e., if these two would give the same answer:
```m2

i16 : netList (table(5,5,i->i),HorizontalSpace=>1,Boxes=>{{0,1,3},{5}})

      -----------------------------------+
o16 = (0, 0) (0, 1) (0, 2) (0, 3) (0, 4) |
      -----------------------------------+
      (1, 0) (1, 1) (1, 2) (1, 3) (1, 4) |
      (2, 0) (2, 1) (2, 2) (2, 3) (2, 4) |
      -----------------------------------+
      (3, 0) (3, 1) (3, 2) (3, 3) (3, 4) |
      (4, 0) (4, 1) (4, 2) (4, 3) (4, 4) |

i17 : netList (table(5,5,i->i),HorizontalSpace=>1,Boxes=>{{0,1,3},{-1}})

      ----------------------------------
o17 = (0, 0) (0, 1) (0, 2) (0, 3) (0, 4)
      ----------------------------------
      (1, 0) (1, 1) (1, 2) (1, 3) (1, 4)
      (2, 0) (2, 1) (2, 2) (2, 3) (2, 4)
      ----------------------------------
      (3, 0) (3, 1) (3, 2) (3, 3) (3, 4)
      (4, 0) (4, 1) (4, 2) (4, 3) (4, 4)
```

Reply:
I thought about negative numbers. but I don't think it makes much arithmetic sense to have the column *after* the last column labelled as "-1". so I'd rather not.

Reply:
okay